CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # MMM-Fuel Changelog
 
+## [2.3.1]
+
+### Added
+
+* Config option `excludeStationIds` to exclude gas stations from radius (Tankerkönig only)
```

Comment:
Can you please add this entry to 2.3.0?

That version is still work in progress and unreleased.

Reply:
Sure thing.